distribution/lib/Standard/Base/0.0.0-dev/src/Data/Text/Normalization.enso

```diff
@@ -0,0 +1,57 @@
+import project.Any.Any
+
+import project.Data.Vector.Vector
+import project.Metadata.Display
+import project.Metadata.Widget
+from project.Metadata.Choice import Option
+from project.Metadata.Widget import Single_Choice
+
+polyglot java import com.ibm.icu.text.Normalizer2.Mode
+
+## Enso representations of `com.ibm.icu.text.Normalizer2.Mode`. Does not
+   include all values of `Mode`.
+type Normalization
+    # Unicode standard NFC
+    NFC
+    # Unicode standard NFD
+    NFD
+    # Unicode standard NFKC
+    NFKC
+    # Unicode standard NFKD
+    NFKD
+    # Unicode standard NFKC_CF
```

Comment:
I'd appreciate some explanation what case folding here involves, although maybe it is not needed since it's an advanced feature so we can expect the user to know/google this.

Reply:
They're not easy to summarize briefly. Following the Java documentation, I just name them clearly and assume anyone using it will know what the mean.